working/augmentation-libraries/feature-specification.md

```diff
@@ -266,6 +266,12 @@ syntactically mark a declaration as an augmentation of an existing one. The
 introduction of this new identifier will be language versioned in order to make
 it non-breaking for old code.
 
+It is also allowed for a non-abstract class to have abstract members, if those
+member bodies are filled in by an augmentation. You should not explicitly mark
+those members as `abstract` or `external`, you can simply leave off the body to
+make them implicitly abstract. This is primarily useful for macros, which may
+be used to provide a body for an abstract member.
```

Comment:
Do we allow abstract field declarations? It was a later syntactic addition to Dart. It looks like:

```dart
abstract class Foo {
  abstract int bar;
}
```

If so, and I think we do, you'll want to reword the paragraph a bit. Maybe just omit the second sentence.